src/utils/statistics/TextBlockchainDatachart.tsx

```diff
@@ -16,39 +16,39 @@ export const TextBlockchainDatachart = ({
     return (
         <>
             <Grid xs={12} md={6}>
-                {typeStatistic === typeBlockchainDataChart.DAILY_TRANSACTIONS && (
+                {typeStatistic === typeChartData.DAILY_TRANSACTIONS && (
                     <Text backgroundColor={isDark ? '#0f172a' : '#F5F6FA'}>
                         Highest number of {dataStatistics.highestValue} transactions on <br />
                         {`${moment(startDate).format('dddd, MMMM DD, YYYY')}`}
                     </Text>
                 )}
-                {typeStatistic === typeBlockchainDataChart.UNIQUE_ADRESSES && (
+                {typeStatistic === typeChartData.UNIQUE_ADRESSES && (
                     <Text backgroundColor={isDark ? '#0f172a' : '#F5F6FA'}>
                         Highest increase of {dataStatistics.highestValue} new addresses was recorded
                         on <br /> {`${moment(startDate).format('dddd, MMMM DD, YYYY')}`}
                     </Text>
                 )}
-                {typeStatistic === typeBlockchainDataChart.ACTIVE_ADDRESSES && (
+                {typeStatistic === typeChartData.ACTIVE_ADDRESSES && (
                     <Text backgroundColor={isDark ? '#0f172a' : '#F5F6FA'}>
                         Highest number of {dataStatistics.highestValue} addresses on <br />
                         {`${moment(startDate).format('dddd, MMMM DD, YYYY')}`}
                     </Text>
                 )}
             </Grid>
             <Grid xs={12} md={6}>
```

Comment:
missing Item in all of the grids in this file `<Grid item ...>`

Reply:
Solved